maxima/g0/derived_geo_calculation/ms-calc-tok-derived-geo.mac

```diff
@@ -0,0 +1,34 @@
+/* Create kernels to project a Maxwellian onto the DG basis using quadrature. */
```

Comment:
wrong comment. Please update the comments in your scripts so they are accurate, otherwise another person won't be able to learn from, modify or extend them.